sdk/keyvault/Azure.Security.KeyVault.Administration/samples/Sample2_RbacScopeAssignment.md

```diff
@@ -0,0 +1,36 @@
+# Creating a Role Assignment for Specific Scopes
+
+By default role assignments apply to the global scope. It is also possible to be more specific by applying an assignment to the all keys scope or a specific `KeyVaultKey`.
+For information about interacting with a `KeyVaultKey` with a `KeyClient`, see the [Key Client README](../../Azure.Security.KeyVault.Keys/README.md) for links and instructions.
+
+## Assigning a Role to the All Keys Scope
+
+Let's assign a role to a service principal soo that it applies to all keys. To do this we'll need a role definition Id and a service principal object Id.
+
+A role definition Id can be obtained from the `Id` property of one of the role definitions returned from `GetRoleAssignments`.
+
+See the [README](../README.md) for links and instructions on how to generate a new service principal and obtain it's object Id.
+You can also get the object Id for your currently signed in account by running the following [Azure CLI][azure_cli] command.
+```
+az ad signed-in-user show --query objectId
+```
+
```

Comment:
You might put something in the middle. When rendered, this will look weird and may not delineate well what is code and what is a command.